dos/src/main.rs

```diff
@@ -234,6 +419,17 @@ fn main() {
     let data_type = value_t_or_exit!(matches, "data_type", String);
     let data_input = value_t!(matches, "data_input", String).ok();
 
+    let transaction_params = match data_type.as_str() {
```

Comment:
I've refactored cmdline interface in such a way that now we use "derive" style for clap (since v3). All the changes are in separate commit https://github.com/solana-labs/solana/pull/23280/commits/90bbd9af9013e894a658180c881a0da774736d5d